subsys/net/lib/lwm2m/lwm2m_rw_json.c

```diff
@@ -453,6 +508,10 @@ static size_t put_bool(struct lwm2m_output_context *out,
 const struct lwm2m_writer json_writer = {
 	put_begin,
```

Comment:
Hmm, I missed this one earlier. Doing it like this is error prone, please change this to use the alternate syntax
```
.put_begin = put_begin,
.put_end = put_end,
...
```
This way the ordering of the fields does not matter and we can safely add new fields into the struct if needed.